src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/framework/checker/Checker.kt

```diff
@@ -310,7 +256,10 @@ private val jacocoToolRegex = """JacocoTool\s*\(\s*["']([^"^']+)["']\s*\)""".toR
  *   - `include(":subproject")`
  *   - `include(':subproject')`
  */
-private val includeRegex = """include\s*\(\s*["']([^"']+)["']\s*\)""".toRegex()
+//private val includeRegex = """include\s*\(\s*["']([^"']+)["']\s*\)""".toRegex()
+private val includeRegex = """\s*include\s*\(.+""".toRegex()
```

Comment:
> `*Subprojects with redefined paths are not supported!*`

Is it still true? It seems you've done some settings.gradle detecting

Reply:
Yes( 
code like `project(':custom:path').projectDir = file('./another/dir')` isn't supported

Reply:
Ok no problem